src/action/index.js

```diff
@@ -73,21 +73,25 @@ observe(store, 'firstStart', async () => {
 });
 
 /**
- * Triggered after the user's password has unlocked the wallet.
+ * Triggered after the user's password has unlocked the wallet
+ * or a user's password has been successfully reset.
  */
 observe(store, 'walletUnlocked', async () => {
-  await nap();
-  await grpc.closeUnlocker();
-  await grpc.initLnd();
+  if (store.walletUnlocked) {
+    await nap();
+    await grpc.closeUnlocker();
+    await grpc.initLnd();
+  }
```

Comment:
Change to `if (!store. walletUnlocked) return;` like below.

Reply:
fixed :)